__init__.py

```diff
@@ -1 +1,2 @@
 from . import models
+from . import report
```

Comment:
- plural
- Directory's `__init__.py` is actually empty

Reply:
In the OCA it says singular.
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#complete-structur
![temp](https://user-images.githubusercontent.com/8931673/168916784-b517499c-82ac-4953-89b7-709e10febbf0.png)
e

Reply:
Done.